nextcord/types/components.py

```diff
@@ -29,6 +29,7 @@
 
 from typing_extensions import NotRequired
 
+from ..channel import ChannelType
```

Comment:
The types.channel.ChannelType would make more sense I think
```suggestion
from .channel import ChannelType
```